packages/dashboard/src/components/ui-contexts.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+export const TooltipContext = React.createContext({
```

Comment:
Do you think that the tooltip is not in the same category as the `notificationBar` on the file `app-context.tsx`

Reply:
hm I put it in a separate category because to me the notifications bar is a result of interaction with RMF (by displaying whenever there is a conflict) and tooltips are purely UI. but we could move it into the `app-context` file, and also include it in the app context provider as per your prev comment!

Reply:
Yeah, the notification bar, in theory, can be used by any component, it's for display a message of error or success, so it just not for the RMF it's a pure UI component. So, it can with the app context too. =)